### ISSUE TITLE
Relics

### ISSUE THREAD

**Issue by PWNG1999** [giero/LGoH-deck-suggester#6]
If you add relics this program will be much more useful. As it stands, even though it is effective, this program is somewhat obsolete because of the relic system contributing so much to power in this game. For me to see my team's complete potential, there should be be relics added.

**Comment by giero** [comment]
You are 100% right, BUT:
1) there is no 'relic codex' from which I could get all the data about them (and I don't have all the relics)
2) it's more than that - this app was created when there were no relics and no Intensity; those makes current game logic far more complicated than then (current calculations are based only on cards stats)
3) adding relics is doable but will definitely increase the calculation time (which may be acceptable in some way)
4) there is no external data source of cards and relics delivered by game creators from which I could get data for my app (I was asking about that, but didn't get any answer ... so it only depends on my spreadsheet) ...  and I'm going to quit playing this game soon, so filling the spreadsheet will depend on the players now
5) adding relics and intensity systems will require a rewrite of huge part of my app from scratch - I didn't realize it will be such a complex solution; however it's doable and will require some time but still ... it will require external source of data because spreadsheet is not enough for that

tl;dr
I could add it (and even more than that), but game creators would have to deliver some data source of all cards/relics available in the game (because no one will provide such huge amount of data to my spreadsheet).
